nym-registration-client/src/builder/config.rs

```diff
@@ -58,8 +58,16 @@ pub struct MixnetClientConfig {
 
     /// The minimum performance of gateways to use.
     pub min_gateway_performance: Option<u8>,
-}
 
+    ///Setting optionally the poisson rate for cover traffic stream
```

Comment:
nit : space before comment
```suggestion
    /// Setting optionally the poisson rate for cover traffic stream
```

Reply:
Fixed